infra/identity/prod/locals.tf

```
@@ -0,0 +1,18 @@
+locals {
+  prefix    = "io"
+  env_short = "p"
+  env       = "prod"
+  location  = "westeurope"
+  project   = "${local.prefix}-${local.env_short}"
+  domain    = "sign"
+
+  repo_name = "io-sign"
+
+  tags = {
+    CostCenter  = "TS310 - PAGAMENTI & SERVIZI"
+    CreatedBy   = "Terraform"
+    Environment = "Prod"
+    Owner       = "IO"
+    Source      ="https://github.com/pagopa/io-sign/blob/main/src/identity/prod" # "https://github.com/pagopa/dx-typescript/blob/main/infra/identity/prod/westeurope" #
```

```suggestion
    Source      = "https://github.com/pagopa/io-sign/blob/main/src/identity/prod"
```

Do not forget to run `terraform fmt -r`